cloudevents/sdk/http_events.py

```diff
@@ -0,0 +1,217 @@
+"""This module maintains HTTP/HTTPS binary cloudevents
```

Comment:
We probably can add the docstring to the class instead of the whole file, as the whole file is just the class.